### ISSUE TITLE
CommandBar item focus order not reset when some item is enabled from disabled state

### ISSUE THREAD

**Issue by b-priyank** [OfficeDev/office-ui-fabric-react#1928]
### Bug Report

- __Package version(s)__: 2.20.2
- __Browser and OS versions__: Chrome, Windows 10

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (No)

Requested priority: (Blocking, High, Normal, Low) 

Products/sites affected: (if applicable)

#### Describe the issue:

When in CommandBar, we have some disabled button, it focuses on enabled button correctly. Ex.,
In following image, save is disabled. When tabbed, it focuses on cancel button correctly.

![image](https://cloud.githubusercontent.com/assets/9213358/26774620/bf37e0fc-49ee-11e7-8630-60ace3ca4d83.png) 
When tabbed -> 
![image](https://cloud.githubusercontent.com/assets/9213358/26774667/f9cb873c-49ee-11e7-89d9-5f0b28f2b981.png)

But when this save button is enabled, tab order still stays same and it focuses on cancel button rather than save button. i.e.,
![image](https://cloud.githubusercontent.com/assets/9213358/26774751/64ecd5fc-49ef-11e7-9f6a-a801e4b00146.png)

#### Actual behavior:

When tabbed to commandBar having some button enabled from disabled state, even though it comes before in tab order, it is not focused. Rather, the button which was already enabled gets focused.

#### Expected behavior:

When tabbed to commandBar or disabled state of ContextualMenuItem is changed, focus should be reset and follow correct tab order.
